systers_portal/templates/meetup/members.html

```diff
@@ -0,0 +1,9 @@
+{% extends "meetup/base.html" %}
+
+{% block meetup_location_page_content %}
+
+  {% include "meetup/snippets/users-grid.html" with users="organizers" user_list=object.organizers.all %}
```

Comment:
This looks much better. May I suggest an improvement?

What if we make the snippet not as a whole grid, but only ONE user and her image. In this way we can reuse it not only in this grid, but in other places too. So I suggest pulling out the piece from `users-grid.html` from lines 7..19 into a smaller snippet and returning the grid code (rows and everything) to this template.